views/tasks/active-officers-details.html

```diff
@@ -158,7 +103,7 @@ <h2 class="govuk-heading-">1 corporate director</h2>
       {% if naturalSecretaryList %}
         {% set officerRows = (officerRows.push([{html: naturalSecretaryHTML}]), officerRows) %}
       {% endif %}
-      {% set officerRows = (officerRows.push([{html: corpSecretaryHTML}]), officerRows) %}
+      {% set officerRows = (officerRows.push([{html: corporateSecretaryHTML}]), officerRows) %}
```

Comment:
I think this also needs the check around it to see if there are any rows to actually show.